src/clarity-angular/datagrid/datagrid.html

```diff
@@ -36,7 +36,7 @@
 
     <!-- Custom placeholder overrides the default empty one -->
     <ng-content select="clr-dg-placeholder"></ng-content>
-    <clr-dg-placeholder *ngIf="!placeholder"></clr-dg-placeholder>
+    <clr-dg-placeholder *ngIf="!placeholder && items.displayed.length == 0"></clr-dg-placeholder>
```

Comment:
@Jinnie: Yes. I tested that case too while working on this fix. I forgot to tag all the issues which submitting the PR